.eslintrc.json

```diff
@@ -1,63 +1,27 @@
 {
-  "env": {
-    "browser": true,
-    "es6": true
-  },
-  "extends": ["airbnb", "prettier"],
-  "plugins": ["react", "jsx-a11y", "prettier"],
-  "parserOptions": {
-    "ecmaVersion": 2015,
-    "sourceType": "module"
-  },
-  "parser": "babel-eslint",
-  "ecmaFeatures": {
-    "modules": true,
-    "arrowFunctions": true,
-    "classes": true
-  },
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint", "react", "jest", "prettier", "jsx-a11y"],
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:jest/recommended",
+    "plugin:prettier/recommended"
+  ],
   "rules": {
-    "arrow-body-style": [2, "as-needed"],
-    "class-methods-use-this": 0,
-    "comma-dangle": [2, "always-multiline"],
-    "import/imports-first": 0,
-    "import/newline-after-import": 0,
-    "import/no-dynamic-require": 0,
-    "import/no-extraneous-dependencies": 0,
-    "import/no-named-as-default": 0,
-    "import/no-unresolved": 2,
-    "import/no-webpack-loader-syntax": 0,
-    "import/prefer-default-export": 0,
-    "indent": [
-      2,
-      2,
-      {
-        "SwitchCase": 1
-      }
-    ],
-    "jsx-a11y/aria-props": 2,
-    "jsx-a11y/heading-has-content": 0,
-    "jsx-a11y/label-has-for": 2,
-    "jsx-a11y/mouse-events-have-key-events": 2,
-    "jsx-a11y/role-has-required-aria-props": 2,
-    "jsx-a11y/role-supports-aria-props": 2,
-    "max-len": 0,
-    "newline-per-chained-call": 0,
-    "no-confusing-arrow": 0,
-    "no-console": 1,
-    "no-use-before-define": 0,
-    "prefer-template": 2,
-    "react/jsx-closing-tag-location": 0,
-    "react/forbid-prop-types": 0,
-    "react/jsx-first-prop-new-line": [2, "multiline"],
-    "react/jsx-filename-extension": 0,
-    "react/jsx-no-target-blank": 0,
-    "react/require-default-props": 0,
-    "react/require-extension": 0,
-    "react/self-closing-comp": 0,
-    "react/sort-comp": 0,
-    "react/jsx-no-undef": ["error", { "allowGlobals": true }],
-    "jsx-a11y/no-static-element-interactions": 0,
-    "jsx-a11y/click-events-have-key-events": 0,
-    "prettier/prettier": "error"
```

Comment:
how these rules being removed and added?

Reply:
are the removed rules included in the `extends`?

Reply:
never mind, as long as the code passed lint I think it's ok to update rules

Reply:
Yeah, I guess the most important rules of them should be included in the `extends` now. We can add other rules progressively in the future.